data/newMigrations/003-service_categories.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex) {
+    return knex.schema.createTable('service_categories', tbl => {
+        tbl.increments('service_category_id')
+            .primary();
```

Comment:
When doing `.increments` knex actually automatically makes it the primary key behind the scenes, so you don't need `.primary()`.